beacon-chain/slasher/service.go

```diff
@@ -14,13 +17,20 @@ var log = logrus.WithField("prefix", "slasher")
 type ServiceConfig struct {
 	Database                db.SlasherDatabase
 	AttestationStateFetcher blockchain.AttestationStateFetcher
+	IndexedAttestationsFeed *event.Feed
+	BeaconBlockHeadersFeed  *event.Feed
 	StateGen                stategen.StateManager
 	SlashingPoolInserter    slashings.PoolManager
+	StateNotifier           statefeed.Notifier
 	HeadStateFetcher        blockchain.HeadFetcher
 }
 
 // Service for running slasher mode in a beacon node.
 type Service struct {
-	params     *Parameters
-	serviceCfg *ServiceConfig
+	params                 *Parameters
+	serviceCfg             *ServiceConfig
+	blksQueue              *blocksQueue
+	attsQueue              *attestationsQueue
+	indexedAttsChan        chan *ethpb.IndexedAttestation
```

Comment:
Will `indexedAttsChan` be used anywhere else other than `receiveAttestations`? 

If not, then I dont think it should be a service struct field